Kitodo/src/test/resources/hibernate.cfg.xml

```diff
@@ -26,7 +26,7 @@
         <property name="hibernate.connection.username">sa</property>
         <property name="hibernate.connection.password"></property>
         <property name="hibernate.connection.url">
-            jdbc:h2:mem:test;DB_CLOSE_DELAY=-1
+            jdbc:h2:mem:test
```

Comment:
Why did you add this change not to the Selenium version of this file?

Reply:
Because in selenium config, we need this property. Otherwise the h2 database may looses its data during the tests

Reply:
Ok, thank you for the explanation.